src/app/profile/profile-card/profile-card.component.ts

```diff
@@ -0,0 +1,182 @@
+import { Component, OnInit, Input } from '@angular/core';
+import { DataStorage } from 'src/app/shared/data-storage.service';
+import { User } from 'src/app/shared/user.model';
+import { Skill } from 'src/app/shared/skill.model';
+import { Router } from '@angular/router';
+import * as Highcharts from 'highcharts';
+import { ChartModel } from 'src/app/shared/ChartModel.model';
+import { NgForm } from '@angular/forms';
+
+@Component({
+  selector: 'app-profile-card',
+  templateUrl: './profile-card.component.html',
+  styleUrls: ['./profile-card.component.css']
+})
+export class ProfileCardComponent implements OnInit {
+  @Input() inputSkillType: string;
+
+  userData : User = {
+    id: 0,
+    userId: " ",
+    password: " ",
+    firstName: " ",
+    lastName: " ",
+    team: " ",
+    genericSkills: [],
+    domainSkills: [],
+    kbcSkills: []
+  }
+  userName : String;
+  userGenericSkills: Skill[] = [];
+  userDomainSkills: Skill[] = [];
+  userKbcSkills: Skill[] = [];
+  userSkillsArray: Skill[] = [];
+  chartOptions: {};
+  maxSkillValue: number = 8;
+  maxChartWidth: number = 300;
+  maxChartHeight: number = 250;
+  chartData : ChartModel[] = [];
+  qpVariable : string;
+  chartTitleText : string;
+  chartCategory : string;
+  cardSubtitle : string; 
+
+  Highcharts = Highcharts;
+
+  constructor(private datastorageService: DataStorage,
+              private router: Router) { }
+
+  ngOnInit() { 
+    this.datastorageService.loggedInUser.subscribe(
+    users=>{
```

Comment:
Spacings.. and place it in same line where subscribe is present.. line 51
users => {